LEAF_Request_Portal/api/index.php

```diff
@@ -176,4 +176,55 @@
     echo $TemplateReportsController->handler($action);
 });
 
+
+$controllerMap->register('emailTemplateFileHistory', function () use ($p_db, $login, $action) {

```

Comment:
For EmailTemplateController and TemplateFileHIstoryController there should only be 1 controller for each, these are all considered duplicates.

Reply:
Have made the changes